components/layouts/AccountLayout.tsx

```diff
@@ -1,20 +1,30 @@
+'use client'
```

Comment:
You may remove this directive. We're using the pages router, so all are client components.

Reply:
updated